runner/runner/main.py

```diff
@@ -149,6 +153,7 @@ def load_tests(config: Mapping[str, Any]) -> None:
     subprocess_run(
         ["xvfb-run", str(runtime_exe_path), *map(str, plugin_paths)],
         env_override=dict(ILLIXR_DATA=str(data_path), ILLIXR_DEMO_DATA=str(demo_data_path), ILLIXR_RUN_DURATION="10"),
+        env_constants=consts_map,
```

Comment:
```suggestion
        env_override=dict(ILLIXR_DATA=str(data_path), ILLIXR_DEMO_DATA=str(demo_data_path), ILLIXR_RUN_DURATION="10", **consts_map),
```
I don't think this merits being a new argument because it is specific to this use case.

Reply:
Likewise for the other invocations.